src/raftstore/store/worker/apply.rs

```diff
@@ -1221,7 +1222,7 @@ impl ApplyDelegate {
     fn load_entries_for_merge(&self, merge: &CommitMergeRequest, apply_index: u64) -> Vec<Entry> {
         // Entries from [first_index, last_index) need to be loaded.
```

Comment:
Please update the comment. `... last_index]`

Reply:
The comment is valid. It's `[first_index, commit_index]`, which is the same as `[first_index, commit_index + 1)`.